migration/sql-files/105-update-root-area-and-iteration-path-field.sql

```diff
@@ -2,6 +2,10 @@
 UPDATE iterations SET path=text2ltree(concat(path, concat('.',replace(cast(id as text), '-', '_')))) WHERE path!='' AND path IS NOT NULL;
 UPDATE areas SET path=text2ltree(concat(path, concat('.',replace(cast(id as text), '-', '_')))) WHERE path!='' AND path IS NOT NULL;
 
+-- drop constraints
```

Comment:
@aslakknutsen @tinakurian  I haven't move these lines far enough.... :/ They need to be the first thing we do before we modify the path.